content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

```diff
@@ -204,60 +174,143 @@ include that specific part of the total quantity.
 
-.. image:: lots/lots-detailed-operations-popup.png
+.. image:: lots/detailed-operations.png
    :align: center
    :alt: Detailed operations popup for source lot number on sales order.
 
-Upon validating the delivery order, a :guilabel:`Traceability` smart button appears. Click the
-:guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which
-includes a :guilabel:`Reference` document, the :guilabel:`Product` being traced, the
-:guilabel:`Date`, and the :guilabel:`Lot/Serial #` assigned.
+.. seealso::
+   :ref:`Traceability report for lot numbers <inventory/product_management/lot-traceability>`
+
+Lot management
+==============
+
+Manage and view existing lot numbers for products in the :guilabel:`Lot/Serial Numbers` dashboard by
+going to :menuselection:`Inventory app --> Products --> Lots/Serial Numbers`.
+
+By default, lot numbers are grouped by product, and selecting the drop-down menu for each product
+displays the existing lot numbers. Select a lot number to :ref:`modify or add details
+<inventory/product_management/edit-lot>` linked to the lot. Lot numbers can also be :ref:`created
+<inventory/product_management/create-new-lot>` from this page.
+
+.. figure:: lots/lot-dashboard.png
+   :align: center
+   :alt: Show the "Lot/Serial Number" dashboard.
+
+   Display lot numbers, grouped by products, on the **Lot/Serial Number** dashboard.
+
+.. _inventory/product_management/edit-lot:
 
-The :guilabel:`Traceability Report` can also include a :guilabel:`Reference` receipt from the
-previous purchase order, if the product quantities shared the same lot number.
+Modify lot
+----------
+
+Clicking a lot from the :guilabel:`Lot/Serial Number` dashboard reveals a separate page where
+additional information can be provided about the lot.
+
+.. tip::
+   Odoo automatically generates a new :guilabel:`Lot/Serial Number` to follow the most recent
+   number. However, it can be edited, by clicking the line under the :guilabel:`Lot/Serial Number`
+   field, and changing the generated number to any desired one.
+
+On the lot number form, the following fields can be modified:
+
+- :guilabel:`Lot/Serial Number`: Change the lot number linked to the :guilabel:`Product`
+- :guilabel:`Internal Reference`: Records an alternative lot/serial number used within the warehouse
+  that differs from the supplier manufacturer's.
+- :guilabel:`Company`: Specify the company where the lot number is available.
+- :guilabel:`Description`: Add extra details about the lot or serial number in this text field.
+
+.. important::
+   The :guilabel:`Product` and :guilabel:`Quantity` fields **cannot** be modified, as the lot
+   numbers are linked with existing stock moves.
+
+.. image:: lots/lot-number.png
+   :align: center
+   :alt: Show the lot number form.
+
+.. seealso::
+   :doc:`Set expiration dates for lots <expiration_dates>`
+
+When all desired configurations are complete, click the :guilabel:`Save` button to save all changes.
```

Comment:
is this step necessary or will Odoo auto save a change like this? might be worth a quick double-check during your revisions

Reply:
you're right! we don't need to click manual save. I'll remove this instruction